backend/src/Controllers/voterRollUtils.ts

```diff
@@ -46,12 +47,12 @@ export async function getOrCreateElectionRoll(req: IRequest, election: Election,
             election_id: String(election.election_id),
             email: req.user?.email ? req.user.email : undefined,
             voter_id: new_voter_id,
-            ip_address: req.ip,
+            ip_hash: hashString(req.ip),
```

Comment:
nit

Could we use the ip_hash variable here?

Reply:
Updated to reuse the variable, but changed the logic some to preserve the behavior. I'm not sure if this is what we want, but currently when we submit a new voter roll there we always include the IP address even if it isn't required for authentication. The ip_hash variable was being set to null if it isn't required so that the PG query knows to ignore it. I updated the ip_hash to be its own variable and the one sent to the the PG querry to require_ip_hash